test/sharness/t0053-dag.sh

```diff
@@ -133,48 +179,47 @@ test_dag_cmd() {
   '
 
   test_expect_success "output looks correct" '
-    echo "{\"data\":\"CAISBGZvbwoYBA==\",\"links\":[]}" > cat_exp &&
+    echo -n "{\"Data\":{\"/\":{\"bytes\":\"CAISBGZvbwoYBA==\"}},\"Links\":[]}" > cat_exp &&
     test_cmp cat_exp cat_out
   '
 
-  test_expect_success "non-canonical cbor input is normalized" '
-    HASH=$(cat ../t0053-dag-data/non-canon.cbor | ipfs dag put --format=cbor --input-enc=raw) &&
```

Comment:
I guess "cbor" used to mean dag-cbor? I'm not sure how "raw" was working here though since non-canon.cbor is _not_ raw, but the CID is correct for dag-cbor. (btw I've verified many of the CIDs against the JS implementations of our codecs too so we have agreement).

Reply:
So the incoming diff replacing these lines, which says `dag-cbor` in name, --format, and --input-enc, that now seems correct, right?  Agree the outgoing diff was confusing and quite likely what I'd call wrong.